cachi2/interface/cli.py

```diff
@@ -148,6 +169,15 @@ def fetch_deps(
             "already have a vendor/ directory (will fail if changes would be made)."
         ),
     ),
+    config_file: Path = typer.Option(
```

Comment:
Should we also extend this parameter to other relevant commands? The Config object is global and potentially affects all of Cachi2's functions.

Reply:
Maybe move it to the top-level callback in that case? Which would mean the user would call `cachi2 --config-file=... fetch-deps` rather than `cachi2 fetch-deps --config-file=...`

https://github.com/containerbuildsystem/cachi2/blob/ed7c47a16d9085f086bb58a4e926b9143a3e273a/cachi2/interface/cli.py#L62-L74

Reply:
Moved to top-level